src/site/content/en/blog/model-viewer/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Introducing the &lt;model-viewer&gt; web component
+subhead: Adding 3D models to a website can be tricky; &lt;model-viewer&gt; is as easy as writing HTML.
+authors:
+  - joemedley
+date: 2019-02-06
+updated: 2020-06-25
+hero: hero.jpg
+alt: A 3D model of a space suit.
+description: |
+  Adding 3D models to a website is tricky for a variety of reasons including
+  the hosting issues and the high bar of 3D programming. That's why we're
+  introducing the &lt;model-viewer&gt; web component to let you use 3D models
+  declaratively.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - 3d
+  - model-viewer
+---
+
+Adding 3D models to a website is tricky. 3D models ideally will be shown in
+a viewer that can work responsively on all browsers including smartphones,
+desktop, or even the new head-mounted displays. The viewer should support
+progressive enhancement for performance and rendering quality. It should support
+use cases on all devices ranging from older, lower-powered smartphones to newer
+devices that support augmented reality. It should stay up to date with current
```

Comment:
```suggestion
devices that support augmented reality. It should stay up-to-date with current
```

Reply:
This is a prepositional phrase, not an adjective. Therefore, no hyphens.